src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -2274,6 +2274,67 @@ public virtual ForbidResult Forbid(AuthenticationProperties properties)
         public virtual ForbidResult Forbid(AuthenticationProperties properties, params string[] authenticationSchemes)
             => new ForbidResult(authenticationSchemes, properties);
 
+        /// <summary>
+        /// Creates a <see cref="ForbidObjectResult"/> (<see cref="StatusCodes.Status403Forbidden"/> by default).
+        /// </summary>
+        /// <param name="value">Response payload.</param>
+        /// <returns>The created <see cref="ForbidObjectResult"/> for the response.</returns>
+        /// <remarks>
+        /// Some authentication schemes, such as cookies, will convert <see cref="StatusCodes.Status403Forbidden"/> to
```

Comment:
These docs are wrong (and so are the ones for the existing overloads), the status code is not set to 403. It calls ForbidAsync which does whatever the auth handler wants. Only Bearer sets 403.